README.md

```diff
@@ -37,11 +37,11 @@ To install this app into your own Django project,
 
         urlpatterns = [
             ...
-            url(r'^admin/', admin.site.urls),
-            url(r'^subjects/', include('subjects.urls')),
-            url(r'^accounts/login/$', auth.views.login,
+            path('admin/', admin.site.urls),
+            path('subjects/', include('subjects.urls')),
+            path('accounts/login/', auth.views.login,
                 {'template_name': 'subjects/login.html'}, name="login"),
-            url(r'^accounts/logout/$', auth.views.logout,
+            path('accounts/logout/', auth.views.logout,
```

Comment:
The `auth.views.login` and `auth.views.logout` is changed in the tests/urls.py, so they should be changed here to match I presume.

Reply:
Ah yeah, I'll update it.